src/Behat/Behat/Context/Annotation/DocBlockHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Context\Annotation;
+
+/**
+ * Helper class for DocBlock parsing
+ */
+class DocBlockHelper
+{
+    /**
+     * Extracts a description from the provided docblock,
+     * with support for multiline descriptions.
+     */
+    public static function extractDescription(string $docBlock): string
```

Comment:
I'd prefer to see this non-static and the helper injected into the class via constructor - this is to allow it to be replaced by an Extension if desired

Reply:
Apologies for the late reply. My development machine was borked and I was to lazy to re-configure it. I did do that after all, so now I could process this. 

I wasn't sure where to define a service for the `DocBlockHelper` so I defined it twice, once for the `ContextExtension` (service ID `context.docblock_helper`) and once for the `DefinitionExtension` (service ID `definition.doc_block_helper`). That way extensions can override one without necessarily also overriding the other. Gives a bit more freedom.

Reply:
With that, I think this MR is ready to be merged now 🙂 

It would be nice to write a rector at some point that can replace annotations with attributes. 
But that wouldn't belong in this MR. I even doubt it belongs in the Behat repository at all.